apps/portal/components/containers/dashboardLayout/elements/SidebarMenuItems/SidebarMenuItem.tsx

```
@@ -9,32 +18,138 @@ interface ISidebarMenuItemProps {
+                      <Chevron className="tw-w-4 tw-h-4 tw-text-gray-500 " />
+                    ) : (
+                      <Chevron className="tw-w-4 tw-h-4 tw-text-gray-500 tw-rotate-180" />
+                    )}
+                  </div>
+                </div>
+              </div>
+            </div>
+            {isOpen && (
+              <div className={styles.listWrapper}>
+                {foldersList.map((v) => (
+                  <div className={styles.title}> {v}</div>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)